sdcm/nemesis.py

```diff
@@ -3515,7 +3529,7 @@ def start_and_interrupt_rebuild_streaming(self):
             delay=1
         )
         ParallelObject(objects=[trigger, watcher], timeout=timeout + 60).call_objects()
-        self.target_node.wait_db_up(timeout=300)
```

Comment:
you just lost the timeout the nemesis has set...

Reply:
add the timeout (but the problem is, there are 3 different "wait" functions, each has a different timeout default and format)